src/schema/v2/homeView/sections/LatestArticles.ts

```diff
@@ -19,5 +20,22 @@ export const LatestArticles: HomeViewSection = {
   },
   requiresAuthentication: false,
 
-  resolver: withHomeViewTimeout(ArticlesConnection.resolve!),
+  resolver: withHomeViewTimeout(async (parent, args, context, info) => {
```

Comment:
Using logic similar to https://github.com/artsy/metaphysics/blob/35189608da893002ec7605c9edfea60e44988d6d/src/schema/v2/homeView/sections/News.ts#L26-L43